resources/views/formfields/multiple_images.blade.php

```diff
@@ -3,8 +3,8 @@
     <?php $images = json_decode($dataTypeContent->{$row->field}); ?>
     @if($images != null)
         @foreach($images as $image)
-            <div class="img_settings_container" data-field-name="{{ $row->field }}">
-                <img src="{{ Voyager::image( $image ) }}" data-image="{{ $image }}" data-id="{{ $dataTypeContent->id }}">
+            <div class="img_settings_container" data-field-name="{{ $row->field }}" style="float:left;padding-right:15px;">
+                <img src="{{ Voyager::image( $image ) }}" data-image="{{ $image }}" data-id="{{ $dataTypeContent->id }}" style="width:200px; height:auto; clear:both; display:block; padding:2px; border:1px solid #ddd; margin-bottom:5px;">
```

Comment:
Using `width: 200px;` will also upscale the image, which will result in bad quality images if you, for example, has with small images to do.
Recommendation: Use `max-width: 200px;`.

Reply:
Right, same problem also on image field.